src/Nfts/NftsNavigator.tsx

```diff
@@ -10,6 +11,21 @@ const Stack = createStackNavigator<NftRoutes>()
 
 export const NftsNavigator = () => {
   const strings = useStrings()
+  const [showSearch, setShowSearch] = useState(false)
+  const [search, setSearch] = useState('')
+
+  const handleCrossPress = () => {
+    if (search.length > 0) {
+      setSearch('')
+    } else {
+      setShowSearch(false)
+    }
+  }
```

Comment:
show search is only adding complexity, IMHO, we should always display, we have a huge warning telling the users that there's no nfts. confirm with design